api/v1alpha1/feature_flags.go

```diff
@@ -21,11 +21,12 @@ import (
 )
 
 const (
-	annotationFeaturePrefix                   = "alpha.operator.dynatrace.com/feature-"
-	annotationFeatureDisableActiveGateUpdates = annotationFeaturePrefix + "disable-activegate-updates"
-	annotationFeatureDisableHostsRequests     = annotationFeaturePrefix + "disable-hosts-requests"
-	annotationFeatureOneAgentMaxUnavailable   = annotationFeaturePrefix + "oneagent-max-unavailable"
-	annotationFeatureEnableMetricsIngest      = annotationFeaturePrefix + "enable-metrics-ingest"
+	annotationFeaturePrefix                            = "alpha.operator.dynatrace.com/feature-"
+	annotationFeatureDisableActiveGateUpdates          = annotationFeaturePrefix + "disable-activegate-updates"
+	annotationFeatureDisableHostsRequests              = annotationFeaturePrefix + "disable-hosts-requests"
+	annotationFeatureOneAgentMaxUnavailable            = annotationFeaturePrefix + "oneagent-max-unavailable"
+	annotationFeatureEnableMetricsIngest               = annotationFeaturePrefix + "enable-metrics-ingest"
+	annotationFeatureEnableInstrumentMissingContainers = annotationFeaturePrefix + "enable-instrument-missing-containers"
```

Comment:
Would probably rename this to something like "enable-webhook-reinvocation-policy"